mkosi/__init__.py

```diff
@@ -5703,9 +5703,9 @@ def parse_args(argv: Optional[Sequence[str]] = None) -> Dict[str, argparse.Names
     # values of positional arguments. Make sure the verb command gets explicitly passed.
     # Insert a -- before the positional verb argument otherwise it might be considered as an argument of
     # a parameter with nargs='?'. For example mkosi -i summary would be treated as -i=summary.
-    for verb in MKOSI_COMMANDS:
+    for verb in Verb:
         try:
-            v_i = argv.index(verb)
+            v_i = argv.index(verb.name)
```

Comment:
Could also use `f"{verb}"`, but this seems more readable? Unless we're insisting on using format strings for this. Let me know which one and why, please